resources/styles/global/loadable.less

```diff
@@ -0,0 +1,29 @@
+@-webkit-keyframes fadeIn { from { opacity: 0; } to { opacity: 1; } }
```

Comment:
This name should probably be local to the file (in case something else declares `fadeIn` to be something different)